### DISCUSSION TITLE
Does narik works with angular 11?

### DISCUSSION THREAD

**Discussion by st-clair-clarke** [NarikMe/narik-angular#14] [Q&A]
Hi,
I am trying to use narik with my angular 11.x.x project. I am following the page at [https://docs.narik.me/04.-ui-inheritance](https://docs.narik.me/04.-ui-inheritance). 

However, it seems that the page need updating ASAP. For example narik is importing from 'narik-material-ui' when the current package is @narik/material-ui.

**Comment by abadakhshan** [comment]
Hello,
Narik works with Angular 11.
But  You are write. Narik documentation need updating and I will do it soon.

You can see an example with Angular 11 [here](https://github.com/NarikMe/narik-material-demo) .

Don't hesitate to contact me if you have any questions.

**Comment by abadakhshan** [comment]
Hello @st-clair-clarke 

The documentation has been updated. You can see it [here](https://docs.narik.me/04.-ui-layout) or [here](https://github.com/NarikMe/narik-angular/wiki/04.-UI-Layout)

Thank you.

**Comment by st-clair-clarke** [comment]
Hello Ali,
Could I get a barebones Stackblitz example of the usage of Narik sections
as it applies to Angular?

Thanks

**Reply by abadakhshan** [reply]
Hello
I created a barebones sample [here](https://github.com/NarikMe/narik-layout-sample), but unfortunately, it seems that **stackblitz** does not support [Angular custom builders](https://angular.io/guide/cli-builder).

You can see it [here ](https://stackblitz.com/github/NarikMe/narik-layout-sample) on **stackblitz**.
